src/main/java/core/mate/academy/model/Bulldozer.java

```diff
@@ -4,10 +4,18 @@
  * Add some custom fields that could be only in Bulldozer
  * Do not remove no-args constructor
  */
+
 public class Bulldozer extends Machine {
+    private int weight;
```

Comment:
Don't forget about access modifiers for your custom fields in `Bulldozer` class. The field `weight` should be private and accessed through getters and setters to encapsulate the data.